process-migration-service/frontend/package.json

```diff
@@ -16,13 +20,12 @@
       "git add"
     ]
   },
-  "keywords": [
-    "react",
-    "webpack",
-    "babel",
-    "minimal",
-    "starter"
-  ],
+  "jest-junit": {
+    "outputDirectory": "../target",
+    "suiteName": "org.kiegroup.processmigration.js.tests",
+    "suiteNameTemplate": "{filename}",
+    "classNameTemplate": "org.kiegroup.processmigration.js.tests.{filename}.{classname}"
+  },
```

Comment:
Can we maybe put this to jest.config.js so it is not scattered over more files?

I have added localy this:

```
                    "jest-junit",
                    {...
                     titleTemplate:...
                     ...
                    "outputDirectory": "../target/jest-junit-reports",
                    "outputName":"TEST-org.kiegroup.processmigration.js.tests.xml",
                    "suiteName": "org.kiegroup.processmigration.js.tests",
                    "suiteNameTemplate": "{filename}",
                    "classNameTemplate": "org.kiegroup.processmigration.js.tests.{filename}.{classname}"
```

Notice also the name and location of the resulting xml file. On Jenkins we filter these jUnit reports by `**/target/*-reports/TEST-*.xml`. I have also let @domhanak know that we should unify it and not leave the default `junit.xml`.

Reply:
Done